docs/source/upgrade-guide/index.md

```diff
@@ -354,6 +354,22 @@ The input field's placeholder text was moved above the buttons.
 Together these changes improve usability both on small screens and in small containers, such as when the widget is in grid block elements.
 
 
+### Renaming of the `constants/Languages` module
+
+`src/constants/Languages.js` has been renamed to `src/constants/Languages.cjs` since in fact, it's a commonJS module.
```

Comment:
We should also add CommonJS to `styles/Vocab/Plone/accept.txt` (sorted) so that Vale does not complain, and also to `plone/documentation`. I'll take care of these.

```suggestion
`src/constants/Languages.js` has been renamed to `src/constants/Languages.cjs` since, in fact, it's a CommonJS module.
```

Reply:
Documentation PR is ready: https://github.com/plone/documentation/pull/1675